client/lib/upgrades/actions/purchases.js

```diff
@@ -23,6 +24,9 @@ function cancelPurchase( purchaseId, onComplete ) {
 }
 
 function cancelAndRefundPurchase( purchaseId, data, onComplete ) {
+	analytics.tracks.recordEvent( 'calypso_purchase_refund', { product_id: data } );
+	analytics.ga.recordEvent( 'Purchase', 'calypso_purchase_refund', `product_id: ${ data }` );
```

Comment:
Well, I console logged `data` and got `1008` (some product's id). But dunno, maybe it can contain something else sometimes too.